src/locales/ko/dialogue.ts

```diff
@@ -305,82 +305,82 @@ export const PGMdialogue: DialogueTranslationEntries = {
   },
   "hex_maniac": {
     "encounter": {
-      1: "I normally only ever listen to classical music, but if I lose, I think I shall try a bit of new age!",
-      2: "I grow stronger with each tear I cry."
+      1: "평소에는 클래식 음악만 들었는데, 지면 뉴에이지도 좀 들어볼까!",
+      2: "눈물을 흘릴 때마다 더 강해지는 것 같아."
     },
     "victory": {
-      1: "Is this the dawning of the age of Aquarius?",
-      2: "Now I can get even stronger. I grow with every grudge."
+      1: "지금이 물병자리 시대의 시작일까?",
+      2: "이제 나는 더 강해지겠지. 모든 원한과 함께 성장하겠어."
     },
     "defeat": {
-      1: "New age simply refers to twentieth century classical composers, right?",
-      2: "Don't get hung up on sadness or frustration. You can use your grudges to motivate yourself."
+      1: "뉴에이지란 단순히 20세기 클래식 작곡가들을 말하는 거, 맞지?",
+      2: "슬픔이나 좌절에 얽매이지 마. 넌 그 원한을 원동력으로 사용할 수 있어."
     }
   },
   "psychic": {
     "encounter": {
-      1: "Hi! Focus!",
+      1: "안녕! 집중해!",
     },
     "victory": {
-      1: "Eeeeek!",
+      1: "에에에에엣!",
     },
   },
   "officer": {
     "encounter": {
-      1: "Brace yourself, because justice is about to be served!",
-      2: "Ready to uphold the law and serve justice on the battlefield!"
+      1: "마음의 준비를 하시죠, 정의가 곧 실행될 거니까요!",
+      2: "법을 지키고 정의를 위해 봉사할 준비가 되었습니다!"
     },
     "victory": {
-      1: "The weight of justice feels heavier than ever…",
-      2: "The shadows of defeat linger in the precinct."
+      1: "정의의 무게가 그 어느 때보다 무겁게 느껴집니다…",
+      2: "패배의 그림자가 관할 경찰서에 남았습니다."
     }
   },
   "beauty": {
     "encounter": {
-      1: "My last ever battle… That's the way I'd like us to view this match…",
+      1: "나의 마지막 배틀… 이 승부를 그렇게 봐주셨으면 좋겠어요…",
     },
     "victory": {
-      1: "It's been fun… Let's have another last battle again someday…",
+      1: "즐거웠어요… 언젠가 또 다른 마지막 승부를 하죠…",
     },
   },
   "baker": {
     "encounter": {
-      1: "Hope you're ready to taste defeat!"
+      1: "패배의 맛을 볼 준비는 됐겠지!"
     },
     "victory": {
-      1: "I'll bake a comeback."
+      1: "재회의 제빵을 해야겠군."
```

Comment:
```suggestion
      1: "실력이든 빵이든, 굽고 나면 단단해지는 법이라네."
```
baker 클래스 트레이너다 보니까 'make a comeback(복귀하다/재기하다)'에서 make를 bake로 바꾼 언어유희를 쓴 것 같은데, 참 살리기가 어렵네요. 나름 초월번역이랍시고 굽어지는 거랑 굽는 거랑 같이 놓고 해 봤는데 어떨까 모르겠네요.

Reply:
> baker 클래스 트레이너다 보니까 'make a comeback(복귀하다/재기하다)'에서 make를 bake로 바꾼 언어유희를 쓴 것 같은데, 참 살리기가 어렵네요. 나름 초월번역이랍시고 굽어지는 거랑 굽는 거랑 같이 놓고 해 봤는데 어떨까 모르겠네요.

이게 더 나은것같아요 ㅋㅋㅋ
제가 문장길이 맞춘다고 이것저것생각하다보니 current sentence가 됐는데
기본적으로 어린이들한테 좋은 느낌을 주는 스크립트가 한글판 번역이기때문에  해당내용 채택하겠습니다